x/staking/keeper/delay_updates.go

```diff
@@ -0,0 +1,30 @@
+package keeper
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
```

Comment:
does it makes sense for this to be const

Reply:
Yes, that would be better.